ephemeral/cl-adapter/templates/ingress.yaml

```diff
@@ -0,0 +1,26 @@
+{{- $name := .Values.name -}}
+{{- $svcPort := .Values.service.port -}}
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    {{- include "cl-adapter.labels" . | nindent 4 }}
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
+spec:
+  rules:
+    - http:
+        paths:
+          - path: /{{ $name }}
+            pathType: Prefix
+            backend:
+              serviceName: adapter-{{ $name }}
+              servicePort: {{ $svcPort }}
+          - path: /{{ $name }}/(.*)
```

Comment:
I'm guessing we need this path route so that any subroutes are still passed to the root level (which in this case is the EA?)